Source/Core/VideoCommon/AVIDump.cpp

```diff
@@ -156,7 +156,7 @@ bool AVIDump::CreateVideoFile()
   s_codec_context->time_base.num = 1;
   s_codec_context->time_base.den = VideoInterface::GetTargetRefreshRate();
   s_codec_context->gop_size = 12;
-  s_codec_context->pix_fmt = g_Config.bUseFFV1 ? AV_PIX_FMT_BGRA : AV_PIX_FMT_YUV420P;
+  s_codec_context->pix_fmt = g_Config.bUseBGRA ? AV_PIX_FMT_BGRA : AV_PIX_FMT_YUV420P;
```

Comment:
How about, use BGRA if supported, otherwise use the first supported one?

On my computer (macOS 10.12.1, ffmpeg 3.2.2 from Homebrew), the codecs MPEG4 (default for AVI), FFV1, H.264, and FLV all have `AV_PIX_FMT_YUV420P` first in `AVCodec.pix_fmts`.